src/components/Logout.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { withRouter, Redirect } from 'react-router';
+import {logout} from '../actions/AuthActions';
+
+/**
+ * renders the logout component
+ * @param props auth state from redux
+ * @constructor
+ */
+const Logout = (props : any) => (
+    <div><Redirect to={{
+        pathname: '/'
```

Comment:
Doesn't log out,  just redirects, with redux this has the same effect because it is not persistent yet

Reply:
is standard feature doesnt have functionality yet, just a setup for now